instruqt-tracks/terraform-cloud-aws-v2/03-centralize-deployments/check-workstation

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -l
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
+cd /root/hashicat-aws
+
+TOKEN=$(grep token /root/.terraform.d/credentials.tfrc.json | cut -d '"' -f4)
```

Comment:
Since we've got this in JSON format, let's extract the token with `jq`.  That will also give a heads-up if the user has accidentally fumbled the syntax in the process of editing the credentials file.

Reply:
Users do not write the credentials file themselves.